.github/workflows/react-native-ci.yml

```
@@ -11,7 +11,26 @@ on:
     paths:
       - 'ReactNative/**'
       - '.github/workflows/react-native-ci.yml'
-
+on:
```

actionlint (1.7.7)

key "on" is duplicated in workflow. previously defined at line:3,col:1

(syntax-check)